site/client/src/main/scala/Ui.scala

```diff
@@ -15,74 +20,86 @@ import model.Exercises._
 import utils.DomHandler._
 
 object UI {
+  val UNSOLVED = "unsolved"
```

Comment:
This may be also modeled with an ADT of case objects for better type safety down the road if these constants are reused.